python/test/test_spacegrouop_type_from_symmetry.py

```diff
@@ -0,0 +1,70 @@
+"""Test of spacegroup_type_from_symmetry."""
+import os
+from pathlib import Path
+from typing import Callable, List
```

Comment:
`typing.List` is deprecated since python 3.9 (https://docs.python.org/3/library/typing.html#typing.List).
Enabling future behavior by
```python
from __future__ import annotations
```
and replacing `List[Path]` to `list[Path]` may reduce future chores.

c.f. https://peps.python.org/pep-0563/#enabling-the-future-behavior-in-python-3-7

Reply:
It's useful. Thanks for your information. I will update using it for the tests.

By the way, I used typehint in `spglib.py` of this PR unintentionally, but I should keep `spglib.py` in old python style for a while.